include/DetectorModule.hh

```diff
@@ -148,11 +148,13 @@ public:
   Property<double, AutoDefault> chipNegativeXExtraWidth;
   Property<double, AutoDefault> chipPositiveXExtraWidth;
   Property<double, AutoDefault> outerSensorExtraLength;
+  Property<double, Default> centralDeadAreaLength;
 
   Property<bool, Default> removeModule;
 
   Property<double, Default> yawAngleFromConfig;
   Property<double, Default> manualRhoCentre;
+  Property<double, Default> offsetForSensors;
```

Comment:
Hi,
it is not immediately clear from this what the offset is supposed to move the sensor by. Can we add a comment here?

Reply:
Sorry for the delay - I've added a comment now. Hope that helps.